README.md

```diff
@@ -36,33 +36,37 @@ You'll need an internet connection to fetch ASnake's dependencies.
+* For Windows: `C:\Users[my user name].archivessnake.yml`
 
-An example .archivessnake.yml file:
+An minimal example of an .archivessnake.yml file:
 
 ```yaml
-baseurl: 'http://localhost:8089'  
-username: 'admin'  
+baseurl: 'http://localhost:8089'
+username: 'admin'
 password: 'admin'
 ```
 
-| **Setting**     | **Description**                                                               | **Default Value**     |
-|-----------------|-------------------------------------------------------------------------------|-----------------------|
-| `baseurl`         | The location (including port if not on port 80) of your archivesspace backend | http://localhost:4567 |
-| `username`        | Username for authorization                                                    | admin                 |
-| `password`        | Password for authorization                                                    | admin                 |
-| `retry_with_auth` | Whether to respond to 403 errors by trying to authorize and retrying          | True                  |
-| `logging_config`  | Hash with various config values for the logging subsystem                     | **see below**         |
+| **Setting**           | **Description**                                                               | **Default Value**       |
+|-----------------------|-------------------------------------------------------------------------------|-------------------------|
+| `baseurl`             | The location (including port if not on port 80) of your archivesspace backend | http://localhost:4567   |
+| `username`            | Username for authorization                                                    | admin                   |
+| `password`            | Password for authorization                                                    | admin                   |
+| `session_token`       | Session token for authorization                                               | **see below**           |
+| `session_header_name` | Header to send session token in                                               | X-ArchivesSpace-Session |
+| `retry_with_auth`     | Whether to respond to 403 errors by trying to authorize and retrying          | True                    |
+| `logging_config`      | Hash with various config values for the logging subsystem                     | **see below**           |
+
+`username`/`password` and `session_token` are mutually exclusive. In a normally configured ArchivesSpace system, you will want to use `username`/`password`. `session_token` allows you to set a fixed value for the session, in case you are sharing a long-lived session amongst several apps, or using an authorization customization that bypasses the ArchivesSpace login route. Examples of this include proxies or SSO plugins.  `session_header_name` lets you customize the header you pass the session in, since some proxies use a different header than `X-ArchivesSpace-Session`.
```

Comment:
It might be worth being more explicit that a config file _can't_ have a username and a session token. A user may expect behavior such as one being used by default, or being able to choose the authorization method while running ArchivesSnake. I don't think either are necessary, but it might make sense to be explicit here.